Training/python/DataLoader.py

```diff
@@ -118,6 +121,8 @@ def get_generator(self, primary_set = True, return_truth = True, return_weights
         def _generator():
 
             finish_counter = 0
+            if show_progress:
+                pbar = tqdm(total = n_batches)
```

Comment:
`n_batches` is not always a reasonable number, e.g: It is set to -1 when we are reading all dataset.

Reply:
Will implement this thanks, it's just a progress bar for when loading the dataset using tensorflow, so will disable it if n_batches<0